src/pudl/metadata/fields.py

```diff
@@ -4443,6 +4508,106 @@
+    },
+    "is_default": {
+        "type": "boolean",
+        "description": "Indicator of whether the technology is default.",
+    },
+    "is_technology_mature": {
+        "type": "boolean",
+        "description": "Indicator of whether the technology is mature.",
+    },
+    "inflation_rate": {
+        "type": "number",
+        "description": (
+            "Rate of inflation. All dollar values are given in 2021 USD, using the Consumer Price "
+            "Index for All Urban Consumers for dollar year conversions where the source "
+            "year dollars do not match 2021."
+        ),
+    },
+    "interest_rate_during_construction_nominal": {
+        "type": "number",
+        "description": (
+            "Also referred to as construction finance cost. Portion of all-in capital cost "
+            "associated with construction period financing. It is a function of construction "
+            "duration, capital fraction during construction, and interest during construction."
+        ),
+    },
+    "interest_rate_calculated_real": {
+        "type": "number",
+        "description": "Calculated real interest rate.",
+    },
+    "interest_rate_nominal": {
+        "type": "number",
+        "description": "Nominal interest rate.",
+    },
+    "rate_of_return_on_equity_calculated_real": {
+        "type": "number",
+        "description": "Calculated real rate of return on equity.",
+    },
+    "rate_of_return_on_equity_nominal": {
+        "type": "number",
+        "description": "Nomial rate of return on equity.",
+    },
+    "tax_rate_federal_state": {
+        "type": "number",
+        "description": (
+            "Combined federal and state tax rate. The R&D model_case_nrelatb holds tax and "
+            "inflation rates constant at assumed long-term values: 21 percent federal tax rate, "
+            "6 percent state tax rate (though actual state tax rates vary), and 2.5 "
+            "percent inflation rate Excludes effects of tax credits. The Market + Policy "
+            "model_case_nrelatb applies federal tax credits and expires them as consistent with "
+            "existing law and guidelines."
+        ),
+    },
+    "capital_recovery_factor": {
+        "type": "number",
+        "description": "Ratio of a constant annuity to the present value of receiving that annuity for a given length of time.",
+    },
+    "debt_fraction": {
+        "type": "number",
+        "description": (
+            "Fraction of capital financed with debt; Debt fraction is assumed financed with equity; "
+            "also referred to as the leverage ratio."
+        ),
+    },
+    "fixed_charge_rate": {
+        "type": "number",
+        "description": (
+            "Amount of revenue per dollar of investment required that must be collected annually "
+            "from customers to pay the carrying charges on that investment."
+        ),
+    },
+    "wacc_nominal": {
```

Comment:
clarification: Is the difference between these two rates that `nominal rate = real rate * expected inflation`?

Reply:
I believe so!